src/components/views/TableCellContextMenu.tsx

```diff
@@ -99,6 +128,7 @@ export const ContextMenu: React.FC<{
   const isCellSelectionMerged = mergeCells(view.state)
   const isCellSelectionSplittable = splitCell(view.state)
   const { rows, columns } = getSelectedCellsCount(view.state)
+  const headerDir = getAddHeaderDirection(view.state)
```

Comment:
Can you extract a method `isInTableHeader` so it can be used like:
`const headerPosition = isInTableHeader(state, pos) ? 'below' : 'above'`

I am wondering if the existing `isHeaderCellSelected` could achieve that already.

Reply:
updated with isHeaderCellSelected